README.md

```diff
@@ -5,7 +5,7 @@
 [![codecov.io](http://codecov.io/github/JuliaMath/FixedPointNumbers.jl/coverage.svg?branch=master)](http://codecov.io/github/JuliaMath/FixedPointNumbers.jl?branch=master)
 
 This library implements fixed-point number types.  A
-[fixed-point number][wikipedia] represents a fractional, or
+[fixed-point number] represents a fractional, or
```

Comment:
I guess you mean the following?

```suggestion
[fixed-point number][fixed-point number] represents a fractional, or
```

Reply:
You don't need it if it's the same string, see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Works if you look at this branch's README in the repo's "home page"